manifest/manifestutils/package_digests.go

```diff
@@ -0,0 +1,68 @@
+package manifestutils
+
+import (
+	"github.com/cashapp/hermit/errors"
+	"github.com/cashapp/hermit/manifest"
+	"github.com/cashapp/hermit/platform"
+	pstate "github.com/cashapp/hermit/state"
+	"github.com/cashapp/hermit/ui"
+)
+
+// PopulateDigests Add missing digests to the manifest file.
+func PopulateDigests(l *ui.UI, state *pstate.State, localManifest *manifest.AnnotatedManifest) error {
+	return PopulateDigestsForVersion(l, state, localManifest, nil)
+}
+
+// PopulateDigestsForVersion Add digest values for a specific version.
+// We want to be able to use this in autoversion and manifest add-digests commands.
+// For autoversion we might have an old and stale version lying around which will fail to download and break the command.
+// To handle that case just calculate hashes for the new version.
```

Comment:
this was added to handle invalid/old versions for autoversioning. Instead of populating all the version digests only try to populate the latest version manifest.